alembic/versions/2024_05_22_18e3b2aba252_add_yield_columns.py

```diff
@@ -0,0 +1,84 @@
+"""2024_05_22_add_yield_columns
```

Comment:
maybe not important but we could update to today's date

Reply:
I think when the script was created is good enough when there is no other migration in between!